cmd/admin-group-enable.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * MinIO Client (C) 2019 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"errors"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var adminGroupEnableCmd = cli.Command{
+	Name:   "enable",
+	Usage:  "Enable a group",
+	Action: mainAdminGroupEnableDisable,
+	Before: setGlobalsFromContext,
+	Flags:  globalFlags,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET GROUPNAME
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Enable group 'allcents':
+     $ {{.HelpName}} myminio allcents
+`,
+}
+
+// checkAdminGroupEnableSyntax - validate all the passed arguments
+func checkAdminGroupEnableSyntax(ctx *cli.Context) {
+	if len(ctx.Args()) != 2 {
+		cli.ShowCommandHelpAndExit(ctx, "enable", 1) // last argument is exit code
+	}
+}
+
+// mainAdminGroupEnableDisable is the handle for "mc admin remove remove" command.
```

Comment:
shouldn't this be "mc admin group enable|disable" command?

Reply:
Yep, done.